media-gfx/kuickshow/kuickshow-0.10.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+KDE_HANDBOOK="true"
```

Comment:
I've checked `CMakeLists.txt` and since DocTools is among the required dependencies in the main KF5 `find_package` call, this should be set to `forceoptional` in order to remove it.

Reply:
I've just checked. When I compile with `USE="-handbook"`, this fails with `forceoptional`, because cmake still tries to use `kdoctools_create_handbook`.
I'll have to adjust `CMakeLists.txt` to make this work.